packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/sagas.ts

```diff
@@ -178,8 +179,15 @@ export default ({ api, coreSagas, networks }) => {
       addExtraStep = true
     }
 
+    const userData = selectors.modules.profile.getUserData(yield select()).getOrElse({
+      dob: ''
+    } as UserDataType)
+    // check is user already entered DOB
+    const isDobEntered = !!userData?.dob && userData.dob !== ''
```

Comment:
Was the second check just for clarity? If `dob` is an empty string, it won't reach what is after the `&&`